coderd/provisionerdserver/acquirer_test.go

```diff
@@ -282,6 +286,9 @@ func TestAcquirer_BackupPoll(t *testing.T) {
 	jobID := uuid.New()
 	err := fs.sendCtx(ctx, database.ProvisionerJob{}, sql.ErrNoRows)
 	require.NoError(t, err)
+	// acquireCall := trapAcquireProvisionerJob.MustWait(ctx)
```

Comment:
this should be trapPoll, I think, and you can't wait on the trap until after you `startAcquire()`